steps/cadence-innovus-signoff/scripts/main.tcl

```diff
@@ -6,6 +6,9 @@
 # Author : Christopher Torng
 # Date   : January 13, 2020
 
+# Change name rules to be case-insensitive
+update_names -nocase
```

Comment:
Haha, I definitely recognize these changes for sky!

Can you update your comment to be more descriptive, maybe "... to guarantee unique names even with case insensitive tools ("N23" and "n23" becomes -> "N23" and "n23_<num>")"

Reply:
Also can you put a newline between the comment and the code?

Reply:
Done